packages/rum/src/longTaskCollection.ts

```diff
@@ -0,0 +1,21 @@
+import { getTimestamp, msToNs } from '@datadog/browser-core'
+import { LifeCycle, LifeCycleEventType } from './lifeCycle'
+import { RumEventCategory, RumLongTaskEvent } from './rum'
+
+export function startLongTaskCollection(lifeCycle: LifeCycle) {
+  lifeCycle.subscribe(LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED, (entry) => {
+    if (entry.entryType === 'longtask') {
```

Comment:
NIT: this could be an early return
```suggestion
    if (entry.entryType !== 'longtask') {
      return
    }
```